.codesandbox/workspace.json

```diff
@@ -0,0 +1,41 @@
+{
```

Comment:
Interesting, do we need to commit this file?

Reply:
No worries :) happy to help. I have removed this file as part of the new review. Please check the latest updates.